snuba/cli/devserver.py

```diff
@@ -147,9 +147,7 @@ def devserver(*, bootstrap: bool, workers: bool) -> None:
                     "--storage=metrics_counters_buckets",
                     "--storage=metrics_distributions_buckets",
                     "--storage=metrics_buckets",
-                    "--storage=metrics_distributions",
-                    "--storage=metrics_counters",
-                    "--storage=metrics_sets",
+                    "--storage=metrics_raw",
```

Comment:
Just to confirm, does this already work end to end in dev with `metrics_raw`?

Reply:
yes, it does

Reply:
I think it's better to hold off on changing this line until https://github.com/getsentry/snuba/pull/2512. I believe devserver is used by some test suites  / Sentry dev env and this may break tests. It's probably not going to be a big deal since things are still behind the `ENABLE_SENTRY_METRICS_DEV` flag but imo it's still better to do after all the tests will pass.